README.md

```diff
@@ -27,9 +30,32 @@ Once you are logged in, click on the "Personal" text on the top right of your br
 1. Click on "+ Create new secret key" and give it a name.
 1. Copy the key and paste it in the .env file in the root directory of this project. Replacing "your key here" 
 
+### Cloud Based Conversation Memory
+Follow the link and follow the instructions in the README to set up the [Griptape-Structure-Chatbot](https://github.com/griptape-ai/griptape-structure-chatbot). 
+This will give you your
+```shell
+GT_STRUCTURE_ID=<your-structure-id>
+LAMBDA_ENPOINT=<your-lambda-endpoint>
+```
 
 ## Running the Demo
 
+If you would like to run the demo locally, then do not define these variables in your .env. 
+```shell
+    GT_STRUCTURE_ID=
+    GT_CLOUD_BASE_URL=
+    LAMBDA_ENDPOINT=
+    GT_CLOUD_API_KEY=
+```
+
+If you would like to run the demo in a managed environment, then you need to define these variables in your .env: 
```

Comment:
nit: Detail what a managed environment is. ex: "If you would like to run the demo in a managed environment like Skatepark or Griptape Cloud"